git-flow/README.md

```diff
@@ -0,0 +1,73 @@
+    - Once completed, the feature branch is merged back into `develop` via a pull request.
+    - Each significant development must be mentioned as a bullet point in the top-section of [**`NEWS.md`**](../NEWS.md) before being pushed to or merged into `develop`, to serve as a change log for the next release.
+- **Hot-fixes** that need to be brought in asap, independently of any other pending development, are carried out in a dedicated branch (of the form `hotfix/<ID>-short-title`) created from `master`. The branch is merged directly back to `master` as a new **patch release**, and must be also merged into `develop` (or possibly an open _release_ branch).
+
+
+## Versioning and Releases {#release}
+
+**SmaRP** uses a [**semantic versioning**](https://semver.org/) scheme bound to the version of the underlying R package. The basic versioning scheme `major.minor.patch` (e.g. `1.1.2`) is reserved for release tagging and the `master` branch (which reflects the most recent release). On the other hand, a fourth _development_ component `-9000` is added for the not-yet-released development happening in the `develop` and _feature_ branches. The package version is updated for the next release (see below) just before the merge into `master` (from `develop` or a _release_ branch). Afterwards, `-9000` is added again to the new version for the future development.
+
+Here we assume that the most recent release is `1.0.0`, hence the version on `develop` is `1.0.0-9000`.
+Releases should only happen from a **stable `develop`**, possibly creating a **_release_ branch** for the release preparation, with a name of the form `release/v<next-release-version>`, e.g. `release/v1.1.0` for a new _minor_ release.
+
+1. **Release preparation**
+    - Consolidate and re-organize the changes in `NEWS.md` (see e.g. Hadley's [recommendations](http://r-pkgs.had.co.nz/release.html#important-files) and [style guide](https://style.tidyverse.org/news.html#news-release)), using the level-3 header `###` for sections if any (nicer rendering in GitHub)
+        - Changes should have been collected in `NEWS.md` already during development
+    - Decide on the **next version** based on whether it is a _patch_, _minor_, _major_ release
+        - For _patch_ changes: `1.0.0-9000` -> `1.0.1` (mainly for hot-fixes)
+        - For _minor_ changes: `1.0.0-9000` -> `1.1.0` (e.g. any change that affects calculations and numbers in the output, minor app refinements or additions, general maintenance)
+        - For _major_ changes: `1.0.0-9000` -> `2.0.0`
+    - Change version number in `NEWS.md` and `DESCRIPTION` files.
+
+(Note: for the remaining steps, a minor release with  `1.1.0` will be used as an example)
+
+2. **Commit and push** all changes with the comment: `1.1.0 release preps` and `closes` lines for all issues mentioned in the `NEWS.md`, e.g.
```

Comment:
```suggestion
2. **Commit and push** all changes with the `1.1.0 release preps` comment and `closes` statements for all issues mentioned in the `NEWS.md`, e.g.
```

Reply:
Both `release preps` and `closes` statements are part of the comment.